src/sgame/sg_bot_ai.cpp

```diff
@@ -247,18 +247,19 @@ AINodeStatus_t BotSelectorNode( gentity_t *self, AIGenericNode_t *node )
 AINodeStatus_t BotSequenceNode( gentity_t *self, AIGenericNode_t *node )
 {
 	AINodeList_t *sequence = ( AINodeList_t * ) node;
-	int i = 0;
+	int start = 0;
```

Comment:
I don't get how this commit changes anything (except by having more explicit variables).
Before, 1st loop started with `i = numNodes - 1`, ended with ` i >= 0 `, and now it seems the same to me?